app/controllers/admin/products_controller.rb

```diff
@@ -1,18 +1,22 @@
 class Admin::ProductsController < Admin::AdminController
   def index
     @products = Product.all.order(:name)
+    @title = "Loose Ends - Admin - Products"
   end
 
   def show
     @product = Product.find(params[:id])
+    @title = "Loose Ends - Admin - Products - " + @product.name
```

Comment:
I think we should use interpolation for this sort of string building
```
@title = "Loose Ends - Admin - Products - #{@product.name}"
```

Reply:
Good call! I've updated it now.